.eslintrc.json

```diff
@@ -0,0 +1,18 @@
+{
+    "env": {
+        "browser": true,
+        "es6": true
+    },
+    "extends": [
+        "google"
+    ],
+    "globals": {
+    },
+    "parserOptions": {
+        "ecmaVersion": 2018,
+        "sourceType": "module"
+    },
+    "rules": {
+    },
+    "ignorePatterns": ["mousetrap.min.js"]
```

Comment:
I'm surprised this is showing up as a diff on this PR, since this is now in master, right? Hmmmm.

Reply:
I screwed up my rebase somehow.  I'm bad, and I feel bad.